lib/matplotlib/colorbar.py

```diff
@@ -226,15 +226,26 @@ class Colorbar:
     mappable : `.ScalarMappable`
         The mappable whose colormap and norm will be used.
 
-        To show the under- and over- value colors, the mappable's norm should
-        be specified as ::
+        To show the colors versus index instead of on a 0-1 scale, set the
+        mappable's norm to ``colors.NoNorm()``.
 
-            norm = colors.Normalize(clip=False)
+    alpha : float
+        The colorbar transparency between 0 (transparent) and 1 (opaque).
+
+    location : None or {'left', 'right', 'top', 'bottom'}
+        Set the colorbar's *orientation* and *ticklocation*. Colorbars on
+        the left and right are vertical, colorbars at the top and bottom
+        are horizontal. The *ticklocation* is the same as *location*, so if
+        *location* is 'top', the ticks are on the top. *orientation* and/or
+        *ticklocation* can be provided as well and overrides the value set by
+        *location*, but there will be an error for incompatible combinations.
 
-        To show the colors versus index instead of on a 0-1 scale, use::
+        .. versionadded:: 3.7
 
-            norm=colors.NoNorm()
+    %(_colormap_kw_doc)s
 
+    Other Parameters
+    ----------------
```

Comment:
It seems like numpydoc is placing the *Attributes* section between *Parameters* and *Other Parameters*, which I find quite confusing.

![grafik](https://github.com/user-attachments/assets/3811ebac-556e-47f6-a9c3-7b2341cd4caa)

This may even be intended as the section order is documented that way https://numpydoc.readthedocs.io/en/latest/format.html#sections

I propose not use *Other Parameters* then here.